source/dub/internal/vibecompat/core/file.d

```diff
@@ -37,6 +37,9 @@ struct RangeFile {
 	{
 		auto sz = this.size;
 		enforce(sz <= size_t.max, "File is too big to read to memory.");
+        if ( sz == 0 ) {
+            return new ubyte[sz];
+        }
```

Comment:
Indentation. Did you use tabs?

Reply:
No, they're spaces. The rest of this file is tabs though.
My local git is showing the diff similarly to here on GH with weird indentation. Vim shows it indented properly in line with everything else.

Should I make them tabs to match the rest of this file?

Reply:
Yeah, dub was initially written by Sönke and thus used tabs and the Vibe.d style, which is why the files are all with tabs. Hence, we just were pragmatic and kept the tabs.

Reply:
Done, switched to tabs.